### PR TITLE
FISH-6044 Create an Example of using gRPC with an EJB

### PR DESCRIPTION
**Pull request by JamesHillyard**
## Description
An Arquillian test version of the gRPC Payara-Example https://github.com/payara/Payara-Examples/pull/162 used to test a simple gRPC service with EJB in Payara

### Blockers
This will block FISH-6068 as this implements adding the gRPC module to Payara for running Payara Samples

### New tests
grpc-ejb module in Payara Samples

### Testing Performed
Ran the sample locally against a clean install of a remote payara server install.

### Testing Environment
Windows 10, JDK 8, Maven 3.6.3

## Documentation
None

## Notes for Reviewers
When reviewing, please consider if the EJB aspects of this demo are effective, useful and there is sufficient use of EJBs.

Please also note adding the grpc-support jar file instead of as a dependency is intentional, as Payara doesn't build gRPC into the server (Intentionally). It needs to be included before the `restart domain` test to take effect and for gRPC support to be tested.

### CODE REVIEW

**Comment by JamesHillyard** [comment]
Jenkins test please

**Comment by JamesHillyard** [comment]
Jenkins test please

**Review by breakponchito — APPROVED** [review]
LGTM

File: appserver/tests/payara-samples/samples/grpc-ejb/src/main/java/fish/payara/samples/grpc/ejbcomponents/StatelessEjb.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * and therefore, elected the GPL Version 2 license, then the option applies
+ * only if the new code is made subject to such option by the copyright
+ * holder.
+ */
+package fish.payara.samples.grpc.ejbcomponents;
+
+import javax.ejb.EJB;
+import javax.ejb.Stateless;
+
+@Stateless
```
Comment (luiseufrasio) [inline appserver/tests/payara-samples/samples/grpc-ejb/src/main/java/fish/payara/samples/grpc/ejbcomponents/StatelessEjb.java:45]: Would it be interesting to have another example with a Statefull EJB
https://jakarta.ee/specifications/enterprise-beans/4.0/jakarta-enterprise-beans-spec-core-4.0.html#a921

**Review by luiseufrasio — APPROVED** [review]

**Pull request merged**